apps/web/src/pages/Profile/ProfilePage/ProfilePage.tsx

```diff
@@ -218,7 +224,28 @@ export const ProfileForm = ({ user }: ProfilePageProps) => {
               id={PAGE_SECTION_ID.ACCOUNT_AND_PRIVACY}
               data-h2-padding-top="base(x3)"
             >
-              <AccountAndPrivacy {...sectionProps} />
+              <Heading
```

Comment:
Maybe I am missing something, but this section seems superfluous now. Why not link Account and privacy settings directly to the account settings page?

## Screenshot
<img width="611" alt="Screen Shot 2024-04-05 at 10 08 51" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/c0e0f1d8-3536-46a3-915f-55d70cbf96fc">

Reply:
Oops, ya I misunderstood thanks 👍 Fixed in [df87f6c](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/df87f6c99c8e150be36982a41abfbca881d802cd)